src/rgw/rgw_admin.cc

```diff
@@ -340,6 +340,7 @@ void usage()
   cout << "   --bucket=<bucket>         Specify the bucket name. Also used by the quota command.\n";
   cout << "   --pool=<pool>             Specify the pool name. Also used to scan for leaked rados objects.\n";
   cout << "   --object=<object>         object name\n";
+  cout << "   --objects-file=<file>     file containing a list of object names to process\n";
```

Comment:
this seems sound and can handle our use case.  I can hazily imagine a world where you give a pool name and maybe a pattern string that would try to restore all matching objects?